apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/http/HttpParameterParser.java

```diff
@@ -115,9 +117,13 @@
 import static com.apollocurrency.aplwallet.apl.core.http.JSONResponses.missing;
 import static org.slf4j.LoggerFactory.getLogger;
 
-
-public final class ParameterParser {
-    private static final Logger LOG = getLogger(ParameterParser.class);
+/**
+ * This class is just a static helper for parameters. It should be removed later cause it uses the CDI static methods.
+ * @deprecated Use {@link RestParametersParser} class instead of this one.
+ */
+@Deprecated
+public final class HttpParameterParser {
```

Comment:
Might be better to add a Util suffix (HttpParameterParserUtil) to state that doesn't have any constructor and this class if full of static methods?

Reply:
renamed